ocs_ci/helpers/ocs_metrics_exporter_helpers.py

```
@@ -0,0 +1,1047 @@
+        "rbd children",
+        "ceph osd blocklist",
+        "ceph fs subvolume",
+        "exec",
+        "command",
+    ]
+
+    cmd = f"oc logs {pod_obj.name} -n {pod_obj.namespace} --since={time_window}"
+    try:
+        logs = exec_cmd(cmd).stdout
+    except CommandFailed:
+        logger.warning(f"Could not retrieve logs for {pod_obj.name}")
```

Ruff (0.15.21)

[warning] Logging statement uses f-string

(G004)